### ISSUE TITLE
[Feature] Add Support for ARM64

### ISSUE THREAD

**Issue by violentmagician** [evanw/esbuild#123]
Are there any plans to add support for ARM64?
Since GO is able to create cross platform libraries, can a build option be added to create an ARM64 version? 
This will allow this and other dependent projects to run on Raspberry PI 4.

**Comment by evanw** [comment]
No plans at the moment. I don't have a device like that so I wouldn't be able to test it, and it doesn't look like GitHub Actions supports it either?

I'm happy to release it with the caveat that it'll be untested and unsupported. Can you build esbuild for ARM64 yourself and verify that it works on the Raspberry PI 4? And also tell me what build command you used? I'm guessing it's something like this:

```
GOOS=linux GOARCH=arm64 go build -o esbuild-arm64 ./cmd/esbuild
```

**Comment by evanw** [comment]
It's going to take me a little while to publish the package. I have once again been blocked by npm's spam detector. I just emailed npm support and I'll close this issue once they resolve it on their end.

**Comment by evanw** [comment]
I just published this in 0.3.7, so it should work now. Let me know if it works for you or not.

**Comment by violentmagician** [comment]
It worked! I was able to install and use Vite to start playing with Vue3.

Thanks for accepting the PR!
